ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -1397,6 +1397,41 @@ def test_tensorflow_log_poisson_loss(
         atol=1e-2,
     )
 
+@handle_frontend_test(
+    fn_tree="tensorflow.nn.l2_loss",
+    dtype_target_inputs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        min_value=0,
+        max_value=1,
+        min_num_dims=1,
+        max_num_dims=5,
+        shared_dtype=True,
+    ),
+    compute_full_loss=st.booleans(),
```

Comment:
This term is not used

Reply:
hey @KareemMAX thanks for the review, i tried do as you said, and as you said, these arguments except for compute_full_loss are unnecessary, also i have addedd docsstring.

Reply:
Not really sure what you mean. But I think it's safe to remove them

```suggestion
```